src/platform-includes/source-context/java.mdx

```diff
@@ -245,15 +245,18 @@ Add the Sentry Maven Plugin to your project by adding the following lines to you
         <plugin>
             <groupId>io.sentry</groupId>
             <artifactId>sentry-maven-plugin</artifactId>
-            <version>{{@inject packages.version('sentry.java.mavenplugin', '0.0.2') }}</version>
+            <version>{{@inject packages.version('sentry.java.maven-plugin', '0.0.2') }}</version>
+            <!-- Required to allow auto-install of Sentry SDK and Integrations -->
+            <extensions>true</extensions>
             <configuration>
                 <!-- for showing output of sentry-cli -->
                 <debugSentryCli>true</debugSentryCli>
 
-                <!-- download the latest sentry-cli and provide path to it here -->
+                <!-- optionally specify the path to sentry-cli -->
                 <!-- download it here: https://github.com/getsentry/sentry-cli/releases -->
                 <!-- minimum required version is 2.17.3 -->
```

Comment:
```suggestion
                <!-- minimum required version is 2.21.2 -->
```

`m` Probably safer to suggest the version that supports org auth tokens for self hosted.